### ISSUE TITLE
Remove .net 6 runtime requirement from dotnet-helix-service

### ISSUE THREAD

**Issue by chcosta** [dotnet/dnceng#2412]
Dotnet-helix-service has a requirement on the .NET 6 runtime because it utilizes secret-manager, which is not yet updated to .NET 8 support.  AFTER https://github.com/dotnet/dnceng/issues/1430 is completed and secret-manager targets .NET 8, we should remove these additional runtime installs from the dotnet-helix-service build.  

Sample .NET 6 runtime install - https://dev.azure.com/dnceng/internal/_git/dotnet-helix-service?path=/eng/deploy.yaml&version=GBmain&line=30&lineEnd=34&lineStartColumn=1&lineEndColumn=23&lineStyle=plain&_a=contents

There are more locations in the repo which also install 6.0 and should be updated.

### Release Note Category
- [ ] Feature changes/additions 
- [ ] Bug fixes
- [x] Internal Infrastructure Improvements
### Release Note Description
Update pipeline YAML to use .NET 8

**Comment by riarenas** [comment]
This issue depends on an issue that is not part of the spike: https://github.com/dotnet/dnceng/issues/2413

@markwilkie @ilyas1974 Should #2413 be moved to the spike, or this issue removed? (With the understanding that this would add at least 2 or 3 weeks to the spike)?

**Comment by ilyas1974** [comment]
I think this can be removed from the spike (Mark can confirm). Our services will not stop working, we just won't be getting updating.

**Comment by riarenas** [comment]
Agreed. I can't find anything that would break underneath us, other than getting more CG alerts. Removing from the Spike.

**Comment by missymessa** [comment]
Not seeing references to .NET6 in dotnet-helix-service anymore. Considering this work complete.

**Comment by missymessa** [comment]
Reopening because the solution doesn't have references to .NET6, but the build YAML does.

**Comment by missymessa** [comment]
PR https://dnceng.visualstudio.com/internal/_git/dotnet-helix-service/pullrequest/49817
